int64 tick_id
int64 next_generation_size
copy_when_uncertain/BeeUpdate[] bee_updates

---

int64 tick_id
copy_when_uncertain/BeeInit[] bee_inits
